test/utils/wasm3_engine.cpp

```diff
@@ -124,7 +124,7 @@ WasmEngine::Result Wasm3Engine::execute(
 {
     unsigned ret_valid;
     uint64_t ret_value;
-    IM3Function function = reinterpret_cast<IM3Function>(func_ref);
+    auto function = reinterpret_cast<IM3Function>(func_ref);  // NOLINT(performance-no-int-to-ptr)
```

Comment:
Why was this needed?

Reply:
This is probably new clang-tidy warning (as I upgraded to clang-tidy-13). Pointer to integer cast blocks some optimizations. https://clang.llvm.org/extra/clang-tidy/checks/performance-no-int-to-ptr.html.